index.html

```diff
@@ -11,13 +11,13 @@
 <body class="body">
 
   <div class="navbar">
-    <div class="my-logo">My Logo</div>
+    <a class="my-logo">My Logo</a>
     <a class="menu-icon"></a>
   </div>
 
   <div class="main">
     <div class="primary-text">I'm Juan Diaz Glad to see you!</div>
-    <div class="secondary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+    <div class="secondary-text">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
     <div class="connect-text">LET'S CONNECT</div>
     <div class="socialmedia-icons">
       <a class="twitter-icon"></a>
```

Comment:
- [ ] This should be wrapped in a `ul` tag with each `a` tag wrapped in an `li` tag since this is a list of items 👍